public/pages/main/Main.tsx

```diff
@@ -15,50 +15,106 @@
 
 import { Switch, Route, RouteComponentProps, Redirect } from 'react-router-dom';
 import React from 'react';
+import { AppState } from '../../redux/reducers';
 import { CreateDetector } from '../createDetector';
-
 import { DetectorList } from '../DetectorsList';
 import { ListRouterParams } from '../DetectorsList/List/List';
-import { PreviewDetector } from '../PreviewDetector/containers/PreviewDetector';
-import { Dashboard } from '../Dashboard/Container/Dashboard';
-import { APP_PATH } from '../../utils/constants';
+// @ts-ignore
+import { EuiSideNav, EuiPage, EuiPageBody, EuiPageSideBar } from '@elastic/eui';
+import { useSelector } from 'react-redux';
+
+enum Navigation {
+  AnomalyDetection = 'Anomaly detection',
+  Dashboard = 'Dashboard',
+  Detectors = 'Detectors',
+}
 
-interface MainProps {}
+enum Pathname {
+  Dashboard = '/dashboard',
+  Detectors = '/detectors',
+}
```

Comment:
nit: move them to public/utils/constants.ts

Reply:
sure, will change it.